### PR TITLE
Feature/ashutosh learn git

### PR DESCRIPTION
**Pull request by ashuzimetrics**
## Summary

(Summarize the story / bug encountered concisely, link to the story / bug )

## What this PR do?

(What this PR aim to achieve? what changes it brings to user or customer? For example, If this is API change please mention curl request/postman request/open API to describe the change)

## Where should the reviewer start?

(Where the reviewer should start. Please guide the reviewer to how to go through the CL to review it more effectively)

## Recommendations on how to test or use this change

(How the changes are tested and how other can use and test it)

## Link to related and dependent issues

(Please mention related issues)

## Does anything need to be done? (ex. Notify someone, deploy sth else) before deploying this change?

(Please mention open points or issues or remaining things)

### CODE REVIEW

File: demo/program1.html
Comment (Shirali01) [inline demo/program1.html:1]: use html tag first on your code

File: demo/program1.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p>
+    this is 1st tutorial
+</p>
+<p>
+    this is 2nd commit
+</p>
```
Comment (OmkarGade06) [inline demo/program1.html:6]: This is applicable

File: demo/program1.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<p>
```
Comment (RiddhiLotlikar) [inline demo/program1.html:1]: Looks good

File: demo/program1.html
Comment (RepoKalyani) [inline demo/program1.html:1]: **Add some code here!!**

**Review by RiddhiLotlikar — APPROVED** [review]

**Pull request merged**